### PR TITLE
React Router & Nav bar

### LINKED ISSUE #5: Getting React Router working for our basic pages.

### PR DESCRIPTION
**Pull request by Mortuie**
Closes #5 

Have implemented navbar and routes/auth routes, now will look to implement redux so we can get shared state rather than messy props passing.

### CODE REVIEW

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,21 +1,19 @@
 import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import Routes from './routes/Routes.js';
```
Comment (vivekimsit) [inline frontend/src/App.js]: Basically you can remove extension from the import.

http://discuss.babeljs.io/t/es6-import-jsx-without-suffix/172

I think `create-react-app` already includes this preset.

File: frontend/src/components/Dummy.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Dummy = () => <div>Dummy Test</div>;
+
+export default Dummy;
```
Comment (vivekimsit) [inline frontend/src/components/Dummy.js]: Ideally if there not more than one export you can remove verbose variable declarations.

It should work

```
export default = () => <div>Dummy Test</div>;
```

**Comment by nikrb** [comment]
looks good to me

File: frontend/src/components/Dummy.js (hunk 2 above)
Comment (Mortuie) [reply frontend/src/components/Dummy.js]: export default () => <div>Dummy Test</div>; works.

**Pull request merged**